openquake/hazardlib/gsim/bindi_2011scaled.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2014-2020 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`BindiEtAl2011scaled`.
+"""
+import numpy as np
+from scipy.constants import g
+
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, PGV, SA
+
+
+class BindiEtAl2011scaled(GMPE):
+    """
+    Implements scaled GMPE developed by D.Bindi, F.Pacor, L.Luzi, R.Puglia,
+    M.Massa, G. Ameri, R. Paolucci and published as "Ground motion
+    prediction equations derived from the Italian strong motion data",
+    Bull Earthquake Eng, DOI 10.1007/s10518-011-9313-z.
+    SA are given up to 2 s.
+    The regressions are developed considering the geometrical mean of the
+    as-recorded horizontal components
```

Comment:
If the only thing you are changing is the table of coefficients you don't need to rewrite the entire model.  It is enough to do the following in the same BindiEtAl2011.py file:

```
class BindiEtAl2011scaled(BindiEtAl2011):
   COEFFS = CoeffsTable(sa_damping=5, table="""\

   """)